message_express.gemspec

```diff
@@ -27,5 +27,4 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest', '~> 5.0'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rr', '~> 1.2'
-  spec.add_development_dependency 'ruby-kafka'
```

Comment:
We should only be loaded by the app that uses this gem. Because they might use Kafka or not. We should not enforce installing it